packages/contentstack-bootstrap/package.json

```diff
@@ -17,7 +17,7 @@
     "test:report": "nyc --reporter=lcov mocha \"test/**/*.test.js\""
   },
```

Comment:
@cs-raj Bootstrap package version is not updated

Reply:
Done